### PR TITLE
Replaced airlock sounds, added forceclose sound

### PR DESCRIPTION
**Pull request by Onyxarias**
## What Does This PR Do

Replaces the sounds for airlock opening and closing, as well as the bolt and unbolt sounds, with sounds ported from Tau Ceti. Also replaces the Airlock Force Open sound, and alters some filenames so that an individual Airlock Force Close sound could be added, both also ported from Tau Ceti.

## Why It's Good For The Game

Our current airlock sounds are old as hell. This makes them more modern, slightly higher quality, and feels generally more immersive.

## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/70349271/c1772b8a-11e1-4e1c-b6b6-df2bee50aea6

## Testing

Open and closed, bolted and unbolted, and forced a bunch of airlocks. New sounds went brrrr.

## Changelog
:cl:
soundadd: Added a new sound effect for forcing closed airlocks
soundadd: Replaced existing airlock sounds with new ported ones
/:cl:

### CODE REVIEW

**Comment by Danchi299** [comment]
airlock opening sounds nice. airlock bolting sounds off?

**Comment by Coolrune206** [comment]
Will take some getting used to but an appreciable increase in quality.

**Comment by Screampuff** [comment]
always loved these sounds, glad they're coming here

**Comment by Onyxarias** [comment]
> image
> 
> It is a soundadd: too, and could be written in a better way so players can understand.

You're right, i've updated it for better clarity! First PR in like, four years, so I'm relearning things

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1267,7 +1267,7 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
 			return 0
 	use_power(360)	//360 W seems much more appropriate for an actuator moving an industrial door capable of crushing people
 	if(forced)
-		playsound(loc, 'sound/machines/airlockforced.ogg', 30, 1)
+		playsound(loc, 'sound/machines/airlockforcedopen.ogg', 30, 1)
```
Comment (1080pCat) [inline code/game/machinery/doors/airlock.dm:1270]:
```suggestion
		playsound(loc, 'sound/machines/airlockforcedopen.ogg', 30, TRUE)
```

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -1316,7 +1316,7 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
 
 	use_power(360)	//360 W seems much more appropriate for an actuator moving an industrial door capable of crushing people
 	if(forced)
-		playsound(loc, 'sound/machines/airlockforced.ogg', 30, 1)
+		playsound(loc, 'sound/machines/airlockforcedclose.ogg', 30, 1)
```
Comment (1080pCat) [inline code/game/machinery/doors/airlock.dm:1319]:
```suggestion
		playsound(loc, 'sound/machines/airlockforcedclose.ogg', 30, TRUE)
```

**Review by Henri215 — APPROVED** [review]
Sounds are looking good to me

**Review by 1080pCat — APPROVED** [review]

**Review by Contrabang — APPROVED** [review]
LGTM

**Pull request merged**